ghzdac/calibrate.py

```diff
@@ -433,6 +497,86 @@ def calibrateDCPulse(cxn,boardname,channel):
     return (datasetNumber(dataset))
 
 
+def calibrateDCPulse_infiniium(cxn, boardname, channel, intorext10mhz):
```

Comment:
`intorext10mhz` is really hard to read. How about `10_MHz_source` or `reference_source`?